src/app/components/loading-spinner/service/terra-loading-spinner.service.ts

```diff
@@ -44,27 +44,28 @@ export class TerraLoadingSpinnerService
             this._isLoading = false;
 
             // to send no change detection run the setTimeout outside of angular
-            this.zone.runOutsideAngular(() =>
-                {
-                    // set timeout to stop the loading-spinner from blinking because of sequential started events
-                    setTimeout(() =>
-                    {
-                        if(!this._isLoading)
-                        {
-                            // run inside angular zone to detect changes from isLoading to false
-                            this.zone.run(() =>
-                            {
-                                this.subscriber.next(this._isLoading);
-                            });
-                        }
-                    }, 100);
-                }
-            );
+            this.zone.runOutsideAngular(this.runOutsideAngular);
```

Comment:
```suggestion
            this.zone.runOutsideAngular(() => this.runOutsideAngular());
```
sonst tritt dieser Fehler auf
<img width="482" alt="bildschirmfoto 2019-01-21 um 16 09 10" src="https://user-images.githubusercontent.com/21104139/51482511-e8d3b800-1d96-11e9-9284-c04a9735eeee.png">